common/src/main/java/com/wynntils/handlers/scoreboard/ScoreboardHandler.java

```diff
@@ -172,26 +172,38 @@ private boolean isScoreboardValid(List<ScoreboardLine> reconstructedScoreboard)
                 .map(ScoreboardLine::line)
                 .orElse(StyledText.EMPTY)
                 .equals(StyledText.fromString("À"))) {
-            return false;
+            // We did not find a new line at the start, so the scoreboard is invalid
+            return List.of();
         }
 
         // 4. Check for segment correctness
+        //    There are 2 error cases here:
+        //       - Fatal error: We find info that makes the current scoreboard invalid
+        //       - "Valid" error: We find an error, but it only makes the current segment invalid, not the scoreboard
```

Comment:
So what happens if we have a truncated part? Is it just removed from the scoreboard, or do we try to use the part that we were given? (I'm not fully following the code here...)

Reply:
If we think it's a half part, we hide it, and dont process it. Usually after a scoreboard reset (/class) it doesn't get shown by wynn either. So I think we are fine.

Reply:
So these half parts, they are just temporary glitches..?

Reply:
Basically, almost like if Wynn doesn't want to intentionally do it, and is just a bug.

Reply:
Ok. Sounds fine then.